src/universal/auth/pages/RegisterPage/RegisterPage.tsx

```diff
@@ -34,7 +34,7 @@ export default class RegisterPage extends Component<any, any> {
 
   public async handleSubmit(data: IUser) {
     if (data && data.email) {
```

Comment:
Does this need to be async? Why is it different than the handleSubmit in LoginPage.tsx?